src/components/AssetInput/MultiAssetPreview/MultiAssetPreview.tsx

```diff
@@ -0,0 +1,59 @@
+import { IconCaretRight, IconSize } from "@foundation/Icon";
+import { useButton } from "@react-aria/button";
+import { useFocusRing } from "@react-aria/focus";
+import { mergeProps } from "@react-aria/utils";
+import { merge } from "@utilities/merge";
+import React, { FC, useRef } from "react";
+import { AssetType } from "../AssetInput";
+import { Assets } from "./Assets";
+
+export type MultiAssetPreviewProps = {
+    assetsAmount?: number;
+    numberOfLocations?: number;
+    assets?: AssetType[];
+    onClick: () => void;
+};
+
+export const MultiAssetPreview: FC<MultiAssetPreviewProps> = ({
```

Comment:
Could you find out if there should be a loading animation in the MultiAssetPreview? There is one now in the AssetInput so I think it would make sense to have the same for the MultiAssetPreview